phe/__about__.py

```diff
@@ -7,13 +7,12 @@
 
 __title__ = "phe"
 __summary__ = "Partially Homomorphic Encryption library for Python"
-__uri__ = "https://github.com/n1analytics/python-paillier"
+__uri__ = "https://github.com/data61/python-paillier"
 
 # We use semantic versioning - semver.org
 __version__ = "1.4.1-dev0"
 
-__author__ = "N1 Analytics developers"
-__email__ = "info@n1analytics.com"
+__author__ = "CSIRO's Data61"
```

Comment:
either add the confidential-computing@data61.csiro.au email address or remove author_email from setup.py.